scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/ModelValidatorContext.java

```diff
@@ -191,10 +244,22 @@ public void updateTaskWithContext(Task task) {
          */
         private Boolean valid;
 
-        public SpELVariables(Map<String, Serializable> variables, Map<String, String> models) {
+        public SpELVariables(Map<String, Serializable> variables, Map<String, String> models, Set<String> groupsNames) {
             this.variables = variables;
             this.models = models;
             this.tempMap = new HashMap<>();
```

Comment:
u can apply LinkedHashMap for tempMap as well

Reply:
ok